Homework4/4a.py

```diff
@@ -0,0 +1,22 @@
+import random
+
+degree = int(input("Enter coefficient - 'c' for: 2x^c + 4x^c-1 + 5x^c-2 = 0: "))
+
+coef=[]
+
+coef = [random.randint(0,101) for i in range(3)]
```

Comment:
```suggestion
coef = [random.randint(0,100) for i in range(3)]
```
А почему коэффициентов только 3?